### PR TITLE
fix(external_api): Fix number of participants in meeting

### PR DESCRIPTION
**Pull request by dudumanbogdan**
- fix number of participants in meeting from all rooms (includes breakout rooms as well)
- add `roomsInfo` API, https://github.com/jitsi/handbook/pull/365

### CODE REVIEW

**Comment by saghul** [comment]
Jenkins please test this please.

File: react/features/base/participants/actions.ts [hunk 1]
```diff
@@ -381,8 +381,14 @@ export function hiddenParticipantLeft(id: string) {
...
  * instance.
- * @param {boolean} isReplaced - Whether the participant is to be replaced in the meeting.
- * @param {boolean} isVirtualScreenshareParticipant - Whether the participant is a virtual screen share participant.
+ * @param {Object} participantLeftProps - Other participant properties.
+ * @typedef {Object} participantLeftProps
+ * @param {boolean} participantLeftProps.isReplaced - Whether the participant is to be replaced in the meeting.
+ * @param {boolean} participantLeftProps.isVirtualScreenshareParticipant - Whether the participant is a
+ * virtual screen share participant.
+ * @param {boolean} participantLeftProps.isFakeParticipant - Whether the participant is
+* a virtual screen share participant.
```
Comment (mihhu) [inline react/features/base/participants/actions.ts:390]: Please fix param description, it seems to be the one from above

File: react/features/shared-video/middleware.any.js [hunk 2]
```diff
@@ -51,7 +52,13 @@ MiddlewareRegistry.register(store => next => action => {
                 if (isSharingStatus(sharedVideoStatus)) {
                     handleSharingVideoStatus(store, value, attributes, conference);
                 } else if (sharedVideoStatus === 'stop') {
-                    dispatch(participantLeft(value, conference));
+                    const videoParticipant = getParticipantById(state, value);
+
+                    dispatch(participantLeft(value, conference, {
+                        isFakeParticipant: videoParticipant?.isFakeParticipant,
+                        isVirtualScreenshareParticipant: videoParticipant?.isFakeParticipant
```
Comment (mihhu) [inline react/features/shared-video/middleware.any.js:59]: Is `isVirtualScreenshareParticipant` needed here?

Comment (dudumanbogdan) [reply react/features/shared-video/middleware.any.js]: I rechecked the flow with SHARED_VIDEO, no we don't need as virtual screenshare participant is on sharing the screen, for share video we only need isFakeParticipant

**Review by mihhu — APPROVED** [review]

**Pull request merged**